client/web/src/site-admin/SiteAdminWebhooksPage.story.tsx

```diff
@@ -112,11 +106,12 @@ export const FiveWebhooksFound: Story = () => (
 
 FiveWebhooksFound.storyName = '5 webhooks found'
 
-function createWebhookMock(kind: ExternalServiceKind, urn: string): WebhookFields {
+function createWebhookMock(name: string, kind: ExternalServiceKind, urn: string): WebhookFields {
     return {
         __typename: 'Webhook',
         createdAt: '',
         id: `webhook-${urn}`,
+        name: name,
```

Comment:
I'm not sure, but I suspect some linter will complain that you can use shorthand syntax here:
```suggestion
        name,
```

Reply:
It didn't, but I'll fix it anyway if it looks better. What exactly is this kind of magic?

Reply:
oh, it did

```
/root/buildkite/build/sourcegraph/client/web/src/site-admin/SiteAdminWebhooksPage.story.tsx
--
  | 114:9  error  Expected property shorthand  object-shorthand
```

Reply:
okay, found it: if the property name is the same as variable which is assigned to this property -- shorthand syntax can be used, neat thing.